test/unit/spec/EnrollCall_spec.js

```diff
@@ -147,6 +161,15 @@ function (Q, Okta, OktaAuth, LoginUtil, Util, Form, Beacon, Expect, $sandbox,
             Util.stopRouter();
           });
       });
+      itp('shows custom back link if features.showCustomizableBackLinkInMFA is true', function () {
```

Comment:
I guess have test for every factor is a nice to have. In any case we need to rebase and add/fix test for hotp factor.

Reply:
I added a test for every factor... all the Enroll*_spec have a test. I will rebase and add it to HOTP as well

Reply:
rebased and changed HOTP test as well.